spec/rspec/mocks/and_call_original_spec.rb

```diff
@@ -143,6 +143,19 @@ def inst.foo; :bar; end
       expect(subclass.new_instance).to be_a(subclass)
     end
 
+    context 'when a class method is stubbed in the superclass' do
+      it 'still works for class methods defined on a superclass' do
```

Comment:
I find this doc string confusing -- it seems to just restate the context (e.g. "class method on a superclass").  Isn't the point of the spec that a subclass can still use `and_call_original` after it's superclass has been stubbed?